lib/private/Security/CSP/ContentSecurityPolicyNonceManager.php

```diff
@@ -65,17 +65,14 @@ public function getNonce(): string {
 	 * Check if the browser supports CSP v3
 	 */
 	public function browserSupportsCspV3(): bool {
-		$browserWhitelist = [
-			Request::USER_AGENT_CHROME,
-			Request::USER_AGENT_FIREFOX,
-			Request::USER_AGENT_SAFARI,
-			Request::USER_AGENT_MS_EDGE,
+		$browserBlacklist = [
```

Comment:
Nit:
```suggestion
		$browserBlocklist = [
```
But we can also get rid of the variable:
```php
public function browserSupportsCspV3(): bool {
	// Modern browsers all supports CSPv3
	return !$this->request->isUserAgent([Request::USER_AGENT_IE]]);
}
```

Reply:
The idea was to keep the variable for later usage.

Reply:
We can easily add one later if needed :)